projects/Magnetosphere/Magnetosphere.cpp

```diff
@@ -158,12 +158,18 @@ namespace projects {
          RP::get(pop + "_Magnetosphere.nVelocitySamples", sP.nVelocitySamples);
 
          RP::get(pop + "_ionosphere.rho", sP.ionosphereRho);
+         RP::get(pop + "_ionosphere.T", sP.ionosphereT);
          RP::get(pop + "_ionosphere.VX0", sP.ionosphereV0[0]);
          RP::get(pop + "_ionosphere.VY0", sP.ionosphereV0[1]);
          RP::get(pop + "_ionosphere.VZ0", sP.ionosphereV0[2]);
-         RP::get(pop + "_ionosphere.taperRadius", sP.ionosphereTaperRadius);
+         RP::get(pop + "_ionosphere.taperInnerRadius", sP.ionosphereTaperInnerRadius);
+         RP::get(pop + "_ionosphere.taperOuterRadius", sP.ionosphereTaperOuterRadius);
+         if(sP.ionosphereTaperInnerRadius > sP.ionosphereTaperOuterRadius) {
+            cerr << "Error: " << pop << "_ionosphere.taperInnerRadius should be <= taperOuterRadius" << endl;
+            abort();
+         }
 
```

Comment:
Great to finally have ionosphere.T!
Perhaps we should add failsafes to both this and the inner taper radius: if they aren't set in the .cfg (i.e. are still zero), then set T to mangetosphere.T and innertaperradius to the ionosphere radius. That way when people re-use older cfg:s things won't break unexpectedly.

Reply:
Like so?

Reply:
still needed: if innertaperradius==0, set it to ionosphereradius

Reply:
I was thinking that we might want to have a partial profile and set the inner radius on purpose < ionosphere.radius. If that's not the case I can force it, sure.

Reply:
well, we can still allow it, as long as the innertaperradius isn't zero. But by setting the default to zero and in that case changing it to our best guess (ionosphere radius), I think we're maintaining compatibility with the previous approach in a sane way.

Reply:
Not quite because of boost aborting upon unknown options (the old <pop>_ionosphere.taperRadius). But I agree with your suggestion, I'll add that.